src/core/tType.ml

```diff
@@ -406,6 +406,7 @@ and module_dep_origin =
 	| MDepFromTyping
 	| MDepFromImport
 	| MDepFromMacro
+	| MDepIgnore
```

Comment:
Hmm, an "ignore" isn't really an origin. The design would be better if we had `let ignore_dep_origin = function | MDepWhatever -> ...` and then use a sensible origin everywhere.